FluidNC/src/Spindles/NowForeverSpindle.cpp

```diff
@@ -188,11 +194,14 @@ namespace Spindles {
             bool running   = false;
             bool direction = false;  // false = cw, true = ccw
 
-            if (response[1] != 0x03)
+            if (response[1] != 0x03) {
                 return false;
+            }
 
             if (response[2] != 2)  // We expect a result length of 2 bytes
+            {
```

Comment:
Opening brace should be on the same line as the if.  Did clang-format not catch that?

Reply:
The comment after the if was messing with the auto format. I moved it above the if and now it formatted properly everywhere.